guides/v3.4.0/components/passing-properties-to-a-component.md

```diff
@@ -54,6 +54,21 @@ known as being "bound"). That is, if the value of `componentProperty`
 changes in the component, `outerProperty` will be updated to reflect that
 change. The reverse is true as well.
 
+In addition to making properties available, actions can be made available
+to components. This allows data to flow back up to its parent. You pass actions
+like this.
+
+```handlebars {data-filename=app/templates/index.hbs}
+  {{button-with-confirmation
+    text="Click here to unsubscribe."
+    onConfirm=(action 'unsubscribe')
```

Comment:
I think it would be good to update the quotes here in the `action` helper, too, to also use the double quoted version as the other string expression in the template examples:

```
(action "unsubscribe")
```